src/utils/components/AddBootableVolumeModal/components/VolumeMetadata/components/InstanceTypeDrilldownSelect/components/DrilldownMenuItem/DrilldownMenuItemWrapper.tsx

```diff
@@ -0,0 +1,49 @@
+import React, { Dispatch, FC, SetStateAction, useState } from 'react';
+
+import HugepagesCheckbox from '@kubevirt-utils/components/HugepagesCheckbox/HugepagesCheckbox';
+
+import { instanceTypeSeriesNameMapper } from '../../utils/constants';
+import { RedHatInstanceTypeSeries } from '../../utils/types';
+import { seriesHasHugepagesVariant } from '../../utils/utils';
+import RedHatInstanceTypeSeriesSizesMenuItems from '../RedHatInstanceTypeSeriesMenu/RedHatInstanceTypeSeriesSizesMenuItem';
+
+import DrilldownMenuItem from './DrilldownMenuItem';
+
+type DrilldownMenuItemWrapperProps = {
+  selected: string;
+  series: RedHatInstanceTypeSeries;
+  setSelected: Dispatch<SetStateAction<string>>;
+};
+
+const DrilldownMenuItemWrapper: FC<DrilldownMenuItemWrapperProps> = ({ series, ...restProps }) => {
+  const { classAnnotation, seriesName, sizes } = series;
+  const { disabled, Icon, seriesLabel } = instanceTypeSeriesNameMapper[seriesName] || {};
+
+  const [isHugepages, setIsHugepages] = useState(false);
```

Comment:
What if i first select the instancetype and then i click on hugepages? Will i have the old non-hugepages instancetype selected anyways? Or the UI will switch to the huge pages instancetype ? Where is the switch in the code?